plugin/pkg/global/extensionvalidation/admission.go

```diff
@@ -252,6 +252,12 @@ func (e *ExtensionValidator) validateShoot(kindToTypesMap map[string]sets.String
 	}
 
 	for i, worker := range spec.Provider.Workers {
+		if worker.CRI != nil {
```

Comment:
Great thanks. Can we also have a unit test, please? :)

Reply:
Done as well